### ISSUE TITLE
Add a new organization to ROR: Vinmec Hai Phong Hospital (Vietnam)

### ISSUE THREAD

**Issue by adambuttrick** [ror-community/ror-updates#38551]
Summary of request: Add a new organization to ROR

Name of organization: Vinmec Hai Phong Hospital*en
Website: https://www.vinmec.com/eng/hospital/vinmec-hai-phong-hospital-1
Domains: 
Link to publications: https://doi.org/10.34172/jnp.2025.27653
Organization type: Healthcare
Wikipedia page:
Wikidata ID:
ISNI ID:
GRID ID:
Crossref Funder ID:
Other names for the organization:
Aliases: Vinmec Haiphong Hospital*en;Vinmec Hai Phong International Hospital*en;Vinmec Haiphong International Hospital*en;Vinmec Hai Phong General Hospital*en
Labels: Bệnh viện Đa khoa Quốc tế Vinmec Hải Phòng*vi
Acronym/abbreviation: 
Related organizations: https://ror.org/012rvtf75 (parent)
City: Hai Phong
Country: Vietnam
Geonames ID: 1581298
Year established: 2018
How will a ROR ID for this organization be used? To identify research affiliations.
Other information about this request: The official site identifies this as a distinct Vinmec hospital opened in 2018; publications use the hospital independently as an affiliation. Follows from https://github.com/ror-community/ror-updates/issues/35061.

**Comment by adambuttrick** [comment]
Funds and employs researchers. Solid affiliation usage. Should be added.

**Comment by adambuttrick** [comment]
Assigned ROR ID https://ror.org/02qpsxt83 in release v2.12.
